imageprocessor/imageprocessor-core/src/test/java/pl/edu/uj/kimage/processingFlow/ImageContainerTest.java

```diff
@@ -0,0 +1,55 @@
+package pl.edu.uj.kimage.processingFlow;
+
+import org.junit.Test;
+import pl.edu.uj.kimage.Image;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ImageContainerTest {
+
+    @Test
+    public void createSampleImageObject() {
+        Image image = new Image(512, 256);
+        assertThat(image.getWidth()).isEqualTo(512);
+        assertThat(image.getHeight()).isEqualTo(256);
+    }
+
+    @Test
+    public void oneChannelOperation() {
```

Comment:
1) The name should include which the scenario is testing whether positive or for example exception throwing. 
2) You didn't test edge cases